hrjobcontract/api/v3/HRJobContract.php

```diff
@@ -110,6 +119,35 @@ function civicrm_api3_h_r_job_contract_getlengthofservice($params) {
   return civicrm_api3_create_success($result, $params);
 }
 
+/**
+ * HRJobContract.getlengthofserviceymd
+ * 
+ * Return an object containing years, months and days of Length of Service
+ * for specific 'contact_id'.
+ * Useful for front-end part of Contact Summary block.
+ * 
+ * @param array $params
+ * @return array API result descriptor
+ * @throws API_Exception
+ */
+function civicrm_api3_h_r_job_contract_getlengthofserviceymd($params) {
```

Comment:
what is the problem of using **civicrm_api3_h_r_job_contract_getlengthofservice** and do the converting inside the angular part as it was before ?